server.js

```diff
@@ -3,27 +3,107 @@ import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
+import Thought from './models/thought'
```

Comment:
This was a nice touch to keep the model in a separate file. Makes the code in server.js look neat and clean 👍